src/linpred.jl

```diff
@@ -126,7 +134,7 @@ function cholesky(p::DensePredChol{T}) where T<:FP
 end
 cholesky!(p::DensePredQR{T}) where {T<:FP} = Cholesky{T,typeof(p.X)}(p.qr.R, 'U', 0)
 
-function delbeta!(p::DensePredChol{T,<:Cholesky}, r::Vector{T}) where T<:BlasReal
+function delbeta!(p::DensePredChol{T,<:Cholesky}, r::Vector{T}; allowrankdeficient::Bool = false) where T<:BlasReal
```

Comment:
No spaces around `=` for consistency with other uses in the file AFAICT.

Reply:
Done.